quality-assurance/user-acceptance-testing.md

```diff
@@ -0,0 +1,41 @@
+## What is User Acceptance Testing
+
+UAT means testing software against its business requirements i.e. does it solve the business problem as intended?
+
+## Who carries out UAT
+
+Business users are best placed to do UAT, as they are aware of the reality of their business, business processes, roles etc.  Sometimes, a tester will act in place of users.
+
+## When does UAT happen
+
+It will typically take place once a functioning system has been delivered, often after system testing.
+
+## Is UAT always necessary
+
+No.  If a project been delivered with an agile process, each increment of the software should have shown to its users during development.  In this case UAT might not be necessary.
+
+UAT is most likely to be required when software has been delivered by external partner following a waterfall process.
+
+# What needs to be tested
+
+The best way of identifying tests is to survey users about the business processes they carry out.  Alternatively, business requirements documents can be used.
+
+# Does UAT need to be scripted
+
+If possible tests should be scripted at a high level, in only enough detail to make the intended outcomes clear.  The aim should be for users to use the software as they would normally do (i.e. to try and carry out business tasks, using any manuals etc. if necessary), rather than following step-by-step instructions.
```

Comment:
i'd say this is a must. as there needs to be some audit trail that due dilligence was performed and it should be used as a regression pack for future improvements / enhancements